src/components/ImageSVG/imageSVGPropTypes.js

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
```

Comment:
Why are we adding new prop types instead of writing TS?

Reply:
I don't think we have other typescript files ready to implement new component in TypeScript @WojtekBoman for confirmation

Reply:
I've migrated ImageSVG component to TS